.claude/skills/bear-sql-quality/SKILL.md

```
@@ -1,95 +1,95 @@
 ```
 
-### 3. 出力の確認
+### 3. Review Output
 
-**クエリ分析リスト:**
-- 各クエリのコスト
-- パフォーマンスレベル
-- 検出された問題
+**Query analysis list:**
+- Cost of each query
+- Performance level
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)